app/views/instructiveness/assessments/show.html.erb

```diff
@@ -9,11 +9,14 @@
         <%= fa_icon 'users' %><b><%= t('.students', course_name: @course.name) %></b>
         <div class='card-header-actions'>
           <span class='pull-right'>
-            <% if @employee.coordinatorships.include?(@course) %>
-              <%= link_to(t('.draft'), draft_given_course_assessment_path(@course, @assessment), class: 'btn btn-warning btn-sm') if @assessment.saved? %>
-              <%= link_to(t('.save'), save_given_course_assessment_path(@course, @assessment), class: 'btn btn-warning btn-sm') if @assessment.draft? && @assessment.fully_graded? %>
+            <% if  @assessment.results_announcement_event&.active_now? %>
```

Comment:
```suggestion
            <% if @assessment.results_announcement_event&.active_now? %>
```

Burada acaba şu şekilde bir metodlar oluştursan daha iyi olmaz mı? Yapalım diye demiyorum sesli düşünüyorum.

```ruby
  @assessment.publishable? # Değerledirmeler yayınlanabilir  bir zaman diliminde anlamında
  @assessment.published?  # Değrlendirmeler yayındandı gibi

```

Reply:
Düzenleme yaparım, tekrar değerlendirsiniz. Yalnız 'publishable?' değil de 'notların girilebilir olması' ile ilgili yapılan işlemler sanki, bu yüzden 'can_be_graded?' gibi bir isim düşünmüştüm.

Reply:
@dilara bu tür isimlendirmelerde `be` hoş durmaz, `can_graded?` belki olabilir ama basitçe `gradable?` niye demiyoruz?